dummy_robot/dummy_map_server/src/dummy_map_server.cpp

```diff
@@ -64,7 +66,10 @@ int main(int argc, char * argv[])
     msg->data[(++center) % (msg->info.width * msg->info.height)] = 100;
     msg->data[(++rhs) % (msg->info.width * msg->info.height)] = 0;
 
-    msg->header.stamp = rclcpp::Time::now();
+    rclcpp::TimeSource ts(node);
+    rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+    ts.attachClock(clock);
+    msg->header.stamp = clock->now();
```

Comment:
@tfoote Can we move all the instantiation outside of the while loop and keep only the call to `clock->now()` here? (Same below)

Reply:
Yes we should do that.